packages/replay-next/components/console/ErrorStackRenderer.tsx

```diff
@@ -37,7 +37,7 @@ function ErrorStackRendererSuspends({
   pauseId: PauseId;
 }) {
   const client = useContext(ReplayClientContext);
-  const stack = getObjectPropertySuspense(client, pauseId, errorObjectId, "stack").value;
+  const stack = objectPropertyCache.read(client, pauseId, errorObjectId, "stack")?.value;
```

Comment:
:speech_balloon: First time seeing usage of the `suspense` package. Part of me wonders if `.read()` should actually be named `.readSuspense()` instead for clarity.  (I suppose the counter-argument is that "using this _with_ actual `<Suspense>` is the default intended use case" :)  )

Interestingly, this now actually _looks_ more like what I remember seeing in the early Suspense demos, syntax-wise.

Reply:
Yeah, I get where you're coming from with this observation. I waffled on the naming a bit, but settled on `read` as a suggestion from someone in the community (initial name was `fetchSuspense`).

I think it's subjective enough that I'll leave it as is for now.